### ISSUE TITLE
Windows AppVM (templated) becomes sluggish after several hours (ram/swap not implicated)

### ISSUE THREAD

**Issue by tonsimple** [QubesOS/qubes-issues#2678]
#### Affected TemplateVMs (e.g., `fedora-23`, if applicable):
Windows App VM (based on an HVM template)

---

### Expected behavior:
Windows 7  to keep performing with more or less stable characteristics as long as RAM and stuff is available 
### Actual behavior:
About 4-5 hours and GUI (non-seamless mode) sometimes becomes sluggish 

Does not appear RAM related (the windows taskmanager reports >3 GB RAM still free)

taskmanager shows qga is having from 35% to 60% CPU use (jumping up and down wildly)

the logfile qga-20170306-071816-2260 contains the following

====

_[20170306.071816.640-2264-W] VchanReceiveBuffer: (000000000042BAC0, msg_xconf): no data, blocking read
[20170306.071816.671-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.102759.261-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.104935.695-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.104948.927-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.104950.482-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.104950.674-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.104951.810-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.104951.897-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.104955.313-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.105032.150-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.105032.394-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.105032.601-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.105034.204-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.105034.503-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.105034.775-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.105035.103-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.105037.295-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.105039.644-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.105040.400-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.105040.593-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.105041.193-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.105113.806-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.105113.947-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.105149.318-2264-W] VchanReceiveBuffer: (000000000042BAC0, msg_motion): no data, blocking read
[20170306.105224.930-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.105225.196-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.105231.408-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.105231.483-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.105238.495-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.105238.675-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.105255.023-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.105255.270-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.105256.049-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.105256.376-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.105256.511-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.105258.696-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.105258.869-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.105318.666-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.105319.333-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.105323.782-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.105329.596-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.105330.159-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.113559.213-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.113638.708-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.113639.283-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.113639.691-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.113640.422-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.113640.487-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.113640.872-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.113652.000-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.123858.982-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.123936.381-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.123936.702-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.123949.033-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.123949.370-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.124036.370-2264-W] VchanReceiveBuffer: (000000000042BAC0, msg_motion): no data, blocking read
[20170306.124039.018-2264-W] VchanReceiveBuffer: (000000000042BAC0, msg_motion): no data, blocking read
[20170306.124141.491-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.124142.024-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.124143.358-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.124256.790-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.124256.878-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.124617.372-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.124619.295-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.124754.801-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.124754.981-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.124758.199-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.124758.464-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.124823.037-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.124851.899-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.124914.688-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.124915.328-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.124946.736-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.124948.071-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.124950.097-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.124950.628-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.124952.484-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.124954.550-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.125000.109-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.125000.911-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.125130.757-2264-W] VchanReceiveBuffer: (000000000042BAC0, msg_motion): no data, blocking read
[20170306.125154.170-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.125155.837-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.125156.329-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.125156.576-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.125343.980-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.125344.229-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.125555.901-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.125556.706-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.125557.263-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.125557.686-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.130748.263-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.130748.691-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.130802.785-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.130802.856-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.130835.012-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.131412.945-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.131437.349-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.131437.629-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.131438.062-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.131438.432-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.131542.224-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.131542.835-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.131602.672-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.131604.439-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.131605.128-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.131605.559-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.131714.577-2264-W] VchanReceiveBuffer: (000000000042BAC0, msg_motion): no data, blocking read
[20170306.131723.610-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.131733.145-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.131946.594-2264-W] VchanReceiveBuffer: (000000000042BAC0, msg_motion): no data, blocking read
[20170306.132744.222-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.132744.597-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.132800.562-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.132801.709-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.132802.110-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.132803.308-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.132805.836-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.132806.602-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.132914.515-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.132914.697-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.133128.127-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.133129.146-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.133143.569-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.133143.729-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.133745.879-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.135322.294-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.135655.196-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.135655.289-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.135918.239-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.135918.939-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.135919.660-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.135920.083-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.135920.083-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.135921.969-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.140324.567-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.140326.417-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.140326.699-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.140326.950-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.140327.096-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.140327.578-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.140328.658-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.140331.070-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.140340.174-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.140341.011-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.140451.383-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.140452.357-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.141310.807-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.144903.360-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.144925.282-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.144925.494-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.144925.782-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.144926.348-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.145038.479-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.145038.542-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.145051.096-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.145051.374-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.145445.049-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.145445.274-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.145503.875-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.145504.125-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.145504.249-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.150025.063-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.150044.739-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.150044.802-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.150048.295-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.150048.713-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.150053.953-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.150054.010-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.150257.179-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.150257.872-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.150258.045-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.150258.209-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.150313.966-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.150314.367-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.150346.606-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.150347.318-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.150642.348-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.150655.221-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.150911.146-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.150912.413-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.150948.852-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.150950.110-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.150954.843-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.150955.440-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.151230.030-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.151230.079-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.151421.277-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.151422.700-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.151504.446-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.151506.327-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.151543.162-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.151543.441-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.151555.520-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.151555.748-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.151558.055-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.151558.230-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.152007.818-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.152007.963-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.152159.654-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.152201.070-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.152203.818-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.152203.898-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.152234.787-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.152234.839-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.152253.270-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.152253.843-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.152302.615-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.152302.960-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.152318.772-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.152319.014-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.152324.320-2264-W] VchanReceiveBuffer: (000000000042BAC0, msg_motion): no data, blocking read
[20170306.152458.961-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.152500.489-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.152515.114-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.152516.685-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.152548.047-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.152549.558-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.152809.186-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.152809.610-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.152810.755-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.152927.434-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.153453.876-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.153456.076-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.153804.061-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.153804.241-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.154016.872-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.154017.025-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.154315.418-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.154318.858-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.154337.256-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.154338.281-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.154351.868-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.154351.964-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.154555.308-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.154555.547-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.154559.511-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.154559.747-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.154651.927-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.154652.345-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.154706.856-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.154707.010-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.154911.055-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.154911.504-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.154911.562-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.154911.679-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.155106.995-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.155107.155-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.155609.291-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.155609.647-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.155610.105-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.155637.914-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.155811.894-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.155951.823-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.160015.269-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.160015.378-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.160136.471-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.160136.651-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.160201.630-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.160201.806-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.160441.033-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.160441.451-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.160441.492-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.160441.595-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.160558.045-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.160558.751-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.160705.527-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.160705.819-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.161112.466-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.161112.497-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.161112.814-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.161113.028-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.161118.751-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.161119.514-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.161636.373-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.161636.626-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.161739.760-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.161740.152-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.161754.097-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.161754.277-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.161822.107-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.161822.300-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.161912.270-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.162341.227-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.162357.192-2264-W] HandleServerData: got unknown msg type 127, ignoring
[20170306.162430.058-2264-W] HandleServerData: got unknown msg type 127, ignoring_

====
### Steps to reproduce the behavior:

Basically, just have a windows AppVM and work with office stuff for a few hours or so

### General notes:

Any idea what  "HandleServerData: got unknown msg type 127" is and why there's so much of it going on?

**Edit:
added on 21.03.2017**

using "switch user" to exit to windows user logon screen and then logging back in makes the sluggishness go away  for at least one hour.
I guess that counts as workaround, even if perhaps not a very good one.

**Comment by andrewdavidwong** [comment]
So, this has nothing (obvious) to do with Windows *Tools*, just Windows as a TemplateBasedHVM? Does it happen with any other (i.e., non-Windows) TemplateBasedHVMs? Any idea whether this affects Windows on vanilla Xen?

**Comment by tonsimple** [comment]
@andrewdavidwong 
Only started happening when I moved from "vanilla" windows HVM to "templated" one
Will try to run office stuff for a few hours on "vanilla" HVM 

Linux template-based VMs not affected (at least there are two separate internet-connected browser VMs running side-by-side my Windows one and they exhibit none of the sluggishness)

Sadly I'm not knowledgeable enough about normal Xen proper to attempt reproducing there.

**Comment by tonsimple** [comment]
Oh, and the 127 thing, what is it? 
It doesn't look healthy to me (google suggests I'm not only one who gets a lot of this in logs tho)

**Comment by tonsimple** [comment]
Seems to not happen on "vanilla" HVM. Even when I drag window over two screens (which tends to increase qga's cpu load) the qga load does not exceed 30% and there is almost no sluggishness.
I'll make something to fiddle around within excel automatically and go afk for a few more hours, but problem seems to be either specific to, or much more severe on, template-based windows 7 VMs, and seems to at least correlate with high CPU consumption by qga.

**Comment by tonsimple** [comment]
Okay, the Windows HVMs become sluggish, too, but much much slower.
The HVM definitely has "lags" now, but not as severe as the templated VM.

AppVMs based on linux running fine all this time.

So it appears to be a general Windows VM issue that for some reason seems more pronounced on template VMs. Weird.

**Comment by andrewdavidwong** [comment]
Thanks for the thorough investigation and reporting!

**Comment by tonsimple** [comment]
@andrewdavidwong 
You're very welcome. I'd do more but unfortunately don't know enough about Xen and stuff...

BTW, one more thing:
Before "onset of sluggishness", if one launches Chrome inside Windows AppVM and loads youtube, then wiggles cursor over the playing video,  no video tearing/lagging happens.

Onset of sluggishness (whether in HVM or templated VM) however, results in that the same actions (launch youtube in Chrome, wiggle cursor over video very fast) causes intense video hiccuping, tearing, and frame loss, which subside a few seconds after cursor stops moving.

This makes differentiating the states somewhat easier than comparing UI lags (which are kind of hard to measure)

if there is anything else I could do to help, let me know.

**Comment by tonsimple** [comment]
@andrewdavidwong 
a few more things to add:

1) I've reproduced this issue on a different machine with a clean qubes+"templated" windows 7 VM install (an AMD based build, so definitely not something up with me/my install/my hardware*)

2) using "switch user" to exit to windows user logon screen and then logging back in makes the sluggishness go away (confirmed via video test described above) for at least one hour.

As far as I understand going to logon screen causes qga.exe restart, so the issue might indeed be with qga.


*I have intel

**Comment by tonsimple** [comment]
Some further additional testing done

Results:

Issue is present both with qubes tools 3.2.1.3  and 3.2.2.3

Issue is affected by disabling/enabling UAC (should be noted that qga.exe that comes with 3.2.1.3 doesn't work at all if UAC is not disabled)

On version 3.2.2.3 issue arises faster if UAC is off (!!!) but is comparatively more severe (I can continue working "with lags" in case UAC is off and barely notice a thing, while if it is ON, the lags become much more noticeable)

I hope very much that it will be resolved one day. 

It is really grinding my gears :(

**Comment by andrewdavidwong** [comment]
This issue is being closed because:

 - This issue has the "Release 3.2 updates" milestone.
 - [Qubes OS 3.2 recently reached end-of-life (EOL).](https://www.qubes-os.org/news/2019/03/28/qubes-3-2-has-reached-eol/)
 - This issue has not been updated in over a year.

If anyone believes that this issue should be reopened, please let us know in a comment here.
